services/worker/src/worker/job_runners/config_info.py

```diff
@@ -0,0 +1,143 @@
+            by a `/`.
+    Returns:
+        `ConfigInfoResponse`: An object with the dataset_info response for requested config.
+    <Tip>
+    Raises the following errors:
+        - [`~job_runners.config_info.PreviousStepStatusError`]
+        `If the previous step gave an error.
+        - [`~job_runners.config_info.PreviousStepFormatError`]
+            If the content of the previous step doesn't have the expected format
+        - [`~job_runners.config_info.MissingInfoForConfigError`]
+            If the dataset info from the parquet export doesn't have the requested dataset configuration
+        - [`~libcommon.dataset.DatasetNotFoundError`]
+            If the dataset does not exist, or if the
+            token does not give the sufficient access to the dataset, or if the dataset is private
+            (private datasets are not supported by the datasets server)
+    </Tip>
+    """
+    logging.info(f"get dataset_info for {dataset=} and {config=}")
+
+    try:
+        response = get_response(kind="/parquet-and-dataset-info", dataset=dataset)
+    except DoesNotExist as e:
+        raise DatasetNotFoundError(
+            "No response found in previous step for this dataset: '/parquet-and-dataset-info'.", e
+        ) from e
+
+    if response["http_status"] != HTTPStatus.OK:
+        raise PreviousStepStatusError(f"Previous step raised an error: {response['http_status']}..")
+
+    try:
+        content = ParquetAndDatasetInfoResponse(
+            parquet_files=response["content"]["parquet_files"], dataset_info=response["content"]["dataset_info"]
+        )
+    except Exception as e:
+        raise PreviousStepFormatError("Previous step did not return the expected content: 'dataset_info'.", e) from e
+
+    if config not in content["dataset_info"]:
+        if not isinstance(content["dataset_info"], dict):
+            raise PreviousStepFormatError(
+                "Previous step did not return the expected content.",
+                TypeError(f"dataset_info should be a dict, but got {type(content['dataset_info'])}"),
+            )
+        raise MissingInfoForConfigError(
+            f"Dataset configuration '{config}' is missing in the dataset info from the parquet export. "
+            f"Available configurations: {', '.join(list(content['dataset_info'])[:10])}"
+            + f"... ({len(content['dataset_info']) - 10})"
+            if len(content["dataset_info"]) > 10
+            else ""
+        )
+    try:
+        config_info = content["dataset_info"][config]
+
+    except Exception as e:
+        raise PreviousStepFormatError("Previous step did not return the expected content.", e) from e
+
+    return ConfigInfoResponse(dataset_info=config_info)
+
+
+class ConfigInfoJobRunner(JobRunner):
+    @staticmethod
+    def get_job_type() -> str:
+        return "config-info"
+
+    @staticmethod
+    def get_job_runner_version() -> int:
+        return 1
```

Comment:
```suggestion
        return PROCESSING_STEP_CONFIG_INFO_VERSION
```
And import it from libcommon constants